docs-kits/kits/behaviour-twin-kit/adoption-view/to-be-considered.md

```diff
@@ -0,0 +1,57 @@
+﻿---
+id: to-be-considered
+title: To Be Considered
+description: Behaviour Twin KIT
+---
+
+<div style={{display:'block'}}>
+  <div style={{display:'inline-block', verticalAlign:'top'}}>
+
+![Behaviour Twin KIT banner](@site/static/img/kit-icons/behaviour-twin-kit-icon-mini.svg)
+
+  </div>
+  <div style={{display:'inline-block', fontSize:17, color:'rgb(255,166,1)', marginLeft:7, verticalAlign:'top', paddingTop:6}}>
+Behaviour Twin KIT
+  </div>
+</div>
+
+## DATA PROTECTION
+
+### GENERAL
+
+Behaviour Twin use cases usually utilize usage data. Therefore, data protection laws must be complied with. In the EU, the General Data Protection Regulation (GDPR) must be observed.
+
+In general, users of data must have obtained consent from the data owner for each type of processing separately. If a data provider gets a request for passing along or processing usage data, it must ensure, that the data owner has given consent for the requested purpose. A automated consent system may be implemented.
+
+Also note that usage data in a vehicle may be generated by other persons (drivers) than the vehicle owner.
+
+### ACCESS DATA BY VIN
+
+Currently, to access usage data of a vehicle from outside the data provider, the vehicle identification number (VIN) would be needed. Until the release of the current KIT version, there is no general legally permissible option for using VINs to obtain usage data. A regulation
+is under discussion.
+
+In individual cases, the usage of a VIN or some generated anonymized number may be possible.
+
+## DATA SOVEREIGNTY
+
+In general, the supply chain is not fully visible to the companies involved. The visibility is at least one up and one down:
+
+![supply chain visibility](assets/supply-chain-visibility.drawio.svg)
+
+To engage additional actors in the supply chain, the framework agreements and other requirements must be propagated through the chain.
```

Comment:
Very broad statement - especially on one up one down - I would prefer that per definition we keep one up one down. Individual agreements are possible, but per design, Catena-X does not foresee breaking out of this guideline.

I know that for BT we had not managed to finalize the predefined use case framework yet, but you could still link here for the general page where those are listed:
https://catena-x.net/en/catena-x-introduce-implement/governance-framework-for-data-space-operations

Reply:
Revised the description.